OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -106,13 +106,19 @@ public class IngameMenuLogic : ChromeLogic
 			{
 				var gameInfoPanel = Game.LoadWidget(world, "GAME_INFO_PANEL", panelRoot, new WidgetArgs()
 				{
-					{ "activePanel", activePanel }
+					{ "activePanel", activePanel },
+					{ "hideMenu", (Action<bool>)RequestHideMenu }
```

Comment:
Is there a reason you defined this as void method and cast it to `Action<bool>` here? Apparently we defined such explicitly in other places: https://github.com/OpenRA/OpenRA/blob/d6436858a90ee3299f379ec827b0065d7183492f/OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs#L83